conans/cli/commands/new.py

```diff
@@ -40,6 +42,24 @@ def new(conan_api, parser, *args):
     if not template_files and not non_template_files:
         raise ConanException("Template doesn't exist or not a folder: {}".format(args.template))
 
+    template_vars = []
```

Comment:
Not related to this line, but I think we talked about providing the list of predefined layouts (hardcoded) in the command help. (Line 15)